indico/modules/events/abstracts/views.py

```diff
@@ -108,12 +108,15 @@ def render_abstract_page(abstract, view_class=None, management=False):
         review_form = build_review_form(abstract, reviewed_for_tracks[0])
     judgment_form = AbstractJudgmentForm(abstract=abstract, formdata=None)
     review_track_list_form = AbstractReviewedForTracksForm(event=abstract.event_new, obj=abstract, formdata=None)
+    convener_tracks = set(get_convener_tracks(abstract.event_new, session.user))
```

Comment:
We use `get_convener_tracks` here only and we don't really care about the order, so maybe it's worth returning a set directly.